app/views/support/BillingPeriod.scala

```diff
@@ -8,6 +8,7 @@ object BillingPeriod {
       case Month => "every month"
       case Quarter => "every 3 months"
       case Year => "every 12 months"
+      case SixMonthsRecurring => "every six months"
```

Comment:
This would look a little odd if auto renew is turned off as it will show they are on "every six months"
![image](https://cloud.githubusercontent.com/assets/7304387/25844582/cf1ac162-34a2-11e7-8d4a-c795e45e6778.png)